{
 "PRed": [
  {
   "PR": {
    "number": 1,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "boost_cpp1740"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 3,
    "state": "closed"
   },
   "data": {
    "bot_rerun": 1665003534.6013575,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "4.11.0"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 4,
    "state": "closed"
   },
   "data": {
    "bot_rerun": 1665003534.6015697,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "4.12.0"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 5,
    "state": "closed"
   },
   "data": {
    "bot_rerun": 1665003534.6016905,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "4.13.0"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": null,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "urdfdom31"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": null,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 3,
    "migrator_version": 0,
    "name": "boost1780"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": null,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "assimp524"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 9,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "4.14.0"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 13,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "assimp525"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 14,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "4.15.1"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "labels": [],
    "number": null,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "assimp531"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "labels": [],
    "number": null,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "boost_cpp_to_libboost"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "__lazy_json__": "pr_json/1799329634.json"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "5.0.0"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  }
 ],
 "bad": false,
 "pinning_version": "2024.03.29.12.36.21",
 "pre_pr_migrator_attempts": {
  "urdfdom40": 1
 },
 "pre_pr_migrator_status": {
  "urdfdom40": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/8887257584\">bot CI job</a>): main: Traceback (most recent call last):\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/auto_tick.py\", line 1267, in _run_migrator\n    migrator_uid, pr_json = run(\n                            ^^^^\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/auto_tick.py\", line 284, in run\n    rerender_msg = rerender_feedstock(feedstock_dir, timeout=900)\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/rerender_feedstock.py\", line 49, in rerender_feedstock\n    return rerender_feedstock_containerized(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/rerender_feedstock.py\", line 102, in rerender_feedstock_containerized\n    data = run_container_task(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/utils.py\", line 240, in run_container_task\n    raise RuntimeError(\nRuntimeError: Error running rerender-feedstock in container - error RuntimeError('Failed to rerender.\\noutput: INFO:conda_smithy.configure_feedstock:Downloading conda-forge-pinning-2024.04.29.13.03.54\\nTraceback (most recent call last):\\n  File \"/opt/conda/envs/cf-scripts/bin/conda-smithy\", line 10, in <module>\\n    sys.exit(main())\\n             ^^^^^^\\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/cli.py\", line 737, in main\\n    args.subcommand_func(args)\\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/cli.py\", line 586, in __call__\\n    self._call(args, args.temporary_directory)\\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/cli.py\", line 589, in _call\\n    configure_feedstock.main(\\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", line 2616, in main\\n    exclusive_config_file, cf_pinning_ver = get_cached_cfp_file_path(\\n                                            ^^^^^^^^^^^^^^^^^^^^^^^^^\\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", line 2437, in get_cached_cfp_file_path\\n    get_cfp_file_path(smithy_cache)\\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", line 2378, in get_cfp_file_path\\n    response.raise_for_status()\\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/requests/models.py\", line 1021, in raise_for_status\\n    raise HTTPError(http_error_msg, response=self)\\nrequests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: https://api.anaconda.org/download/conda-forge/conda-forge-pinning/2024.04.29.13.03.54/noarch/conda-forge-pinning-2024.04.29.13.03.54-hd8ed1ab_0.conda\\n\\n') raised:\ncmd: ['docker',\n 'run',\n '-e',\n 'CF_TICK_IN_CONTAINER=true',\n '--security-opt=no-new-privileges',\n '--read-only',\n '--cap-drop=all',\n '--mount',\n 'type=tmpfs,destination=/tmp,tmpfs-mode=1777,tmpfs-size=100000000',\n '-m',\n '2048m',\n '--cpus',\n '1',\n '--ulimit',\n 'nofile=1024:1024',\n '--ulimit',\n 'nproc=2048:2048',\n '--rm',\n '-i',\n '--mount',\n 'type=bind,source=/tmp/tmpusvej_6q,destination=/cf_tick_dir',\n 'ghcr.io/regro/conda-forge-tick:master',\n '/opt/conda/envs/cf-scripts/bin/python',\n '-u',\n '/opt/autotick-bot/docker/run_bot_task.py',\n 'rerender-feedstock',\n '--timeout',\n '900',\n '--log-level',\n 'info']\nstdout: {'container_stdout': '',\n 'data': None,\n 'error': \"RuntimeError('Failed to rerender.\\\\noutput: \"\n          'INFO:conda_smithy.configure_feedstock:Downloading '\n          'conda-forge-pinning-2024.04.29.13.03.54\\\\nTraceback (most recent '\n          'call last):\\\\n  File \"/opt/conda/envs/cf-scripts/bin/conda-smithy\", '\n          'line 10, in <module>\\\\n    sys.exit(main())\\\\n             '\n          '^^^^^^\\\\n  File '\n          '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/cli.py\", '\n          'line 737, in main\\\\n    args.subcommand_func(args)\\\\n  File '\n          '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/cli.py\", '\n          'line 586, in __call__\\\\n    self._call(args, '\n          'args.temporary_directory)\\\\n  File '\n          '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/cli.py\", '\n          'line 589, in _call\\\\n    configure_feedstock.main(\\\\n  File '\n          '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", '\n          'line 2616, in main\\\\n    exclusive_config_file, cf_pinning_ver = '\n          'get_cached_cfp_file_path(\\\\n                                            '\n          '^^^^^^^^^^^^^^^^^^^^^^^^^\\\\n  File '\n          '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", '\n          'line 2437, in get_cached_cfp_file_path\\\\n    '\n          'get_cfp_file_path(smithy_cache)\\\\n  File '\n          '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", '\n          'line 2378, in get_cfp_file_path\\\\n    '\n          'response.raise_for_status()\\\\n  File '\n          '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/requests/models.py\", '\n          'line 1021, in raise_for_status\\\\n    raise '\n          'HTTPError(http_error_msg, '\n          'response=self)\\\\nrequests.exceptions.HTTPError: 503 Server Error: '\n          'Service Unavailable for url: '\n          \"https://api.anaconda.org/download/conda-forge/conda-forge-pinning/2024.04.29.13.03.54/noarch/conda-forge-pinning-2024.04.29.13.03.54-hd8ed1ab_0.conda\\\\n\\\\n')\",\n 'timeout': 900,\n 'traceback': 'Traceback (most recent call last):\\n'\n              '  File \"/opt/autotick-bot/docker/run_bot_task.py\", line 118, in '\n              '_run_bot_task\\n'\n              '    data = func(**kwargs)\\n'\n              '           ^^^^^^^^^^^^^^\\n'\n              '  File \"/opt/autotick-bot/docker/run_bot_task.py\", line 241, in '\n              '_rerender_feedstock\\n'\n              '    msg = rerender_feedstock_local(fs_dir, **kwargs)\\n'\n              '          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\\n'\n              '  File '\n              '\"/opt/autotick-bot/conda_forge_tick/rerender_feedstock.py\", '\n              'line 239, in rerender_feedstock_local\\n'\n              '    raise RuntimeError(f\"Failed to rerender.\\\\noutput: '\n              '{ret.stdout}\\\\n\")\\n'\n              'RuntimeError: Failed to rerender.\\n'\n              'output: INFO:conda_smithy.configure_feedstock:Downloading '\n              'conda-forge-pinning-2024.04.29.13.03.54\\n'\n              'Traceback (most recent call last):\\n'\n              '  File \"/opt/conda/envs/cf-scripts/bin/conda-smithy\", line 10, '\n              'in <module>\\n'\n              '    sys.exit(main())\\n'\n              '             ^^^^^^\\n'\n              '  File '\n              '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/cli.py\", '\n              'line 737, in main\\n'\n              '    args.subcommand_func(args)\\n'\n              '  File '\n              '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/cli.py\", '\n              'line 586, in __call__\\n'\n              '    self._call(args, args.temporary_directory)\\n'\n              '  File '\n              '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/cli.py\", '\n              'line 589, in _call\\n'\n              '    configure_feedstock.main(\\n'\n              '  File '\n              '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", '\n              'line 2616, in main\\n'\n              '    exclusive_config_file, cf_pinning_ver = '\n              'get_cached_cfp_file_path(\\n'\n              '                                            '\n              '^^^^^^^^^^^^^^^^^^^^^^^^^\\n'\n              '  File '\n              '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", '\n              'line 2437, in get_cached_cfp_file_path\\n'\n              '    get_cfp_file_path(smithy_cache)\\n'\n              '  File '\n              '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", '\n              'line 2378, in get_cfp_file_path\\n'\n              '    response.raise_for_status()\\n'\n              '  File '\n              '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/requests/models.py\", '\n              'line 1021, in raise_for_status\\n'\n              '    raise HTTPError(http_error_msg, response=self)\\n'\n              'requests.exceptions.HTTPError: 503 Server Error: Service '\n              'Unavailable for url: '\n              'https://api.anaconda.org/download/conda-forge/conda-forge-pinning/2024.04.29.13.03.54/noarch/conda-forge-pinning-2024.04.29.13.03.54-hd8ed1ab_0.conda\\n'\n              '\\n'\n              '\\n'}\n"
 },
 "smithy_version": "3.34.0"
}